src/components/Templates/Toaster/Toaster.component.tsx

```diff
@@ -0,0 +1,32 @@
+
+import { ToastTitle } from "../../Molucules/Toast/ToastTitle"
+import { ToastViewport } from "../../Molucules/Toast/ToastViewport"
+import { ToastClose } from "../../Molucules/Toast/ToastClose"
+import { Toast } from "../../Molucules/Toast/Toast"
+import { useToast } from "../../../config/useToast"
+import { ToastDescription } from "../../Molucules/Toast/ToastDescription"
+import { ToastProvider } from "../../Organisms/ToastProvider"
+   
+export function Toaster() {
+    const { toasts } = useToast()
```

Comment:
@siroshan This is an organism isn't it? Template are usually a section of a page design/layout or  a page itself—but this is an independent component!